packages/api/lib/publishSnsMessageUtils.js

```diff
@@ -6,13 +6,19 @@ const Logger = require('@cumulus/logger');
 
 const logger = new Logger({ sender: '@cumulus/publishSnsMessageUtils' });
 
+const publishPdrSnsMessage = async (record) => {
+  logger.info(`About to publish SNS message for pdr to topic ARN ${process.env.pdr_sns_topic_arn}: ${JSON.stringify(record)} `);
+  const topicArn = envUtils.getRequiredEnvVar('pdr_sns_topic_arn', process.env);
+  await publishSnsMessage(topicArn, record);
+};
+
 const publishExecutionSnsMessage = async (record) => {
   const topicArn = envUtils.getRequiredEnvVar('execution_sns_topic_arn', process.env);
-  logger.info(`About to publish SNS message ${JSON.stringify(record)} for execution to topic ARN ${topicArn}`);
+  logger.info(`About to publish SNS message for execution to topic ARN ${topicArn}: ${JSON.stringify(record)} `);
   await publishSnsMessage(topicArn, record);
 };
 
-const constructSnsMessage = (record, event) => {
+const constructCollectionSnsMessage = (record, event) => {
```

Comment:
I assume this means we are only and will only be using it for collections?

Reply:
Actually, working on the granules portion, I may refactor this because it seems like granule message construction looks the same!

Reply:
I ended up reverting the change and am going to leave it as is because the `REMOVE` event for granules generates the message using the entire oldImage whereas collections just uses name and version 

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lambdas/publish-granules.js#L29-L36

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lambdas/publish-collections.js#L16